hawc/apps/lit/models.py

```diff
@@ -815,5 +815,29 @@ def get_hero_id(ref):
                 return int(ident.unique_id)
         return None
 
+    def update_from_hero_content(self):
```

Comment:
duplicates `https://github.com/shapiromatron/hawc/blob/master/hawc/apps/lit/models.py#L590-L604`; pull this code out and into something that works in both places. Something like:

```python
def update_from_hero_content(self, hero_content:Dict):
   ...
```
This would take the json.loaded content from the identifier as an input, and it wouldn't save by default, you could have a save flag if you'd like that's commonly done in djangoland.